osm2streets/src/road.rs

```diff
@@ -115,8 +114,8 @@ impl Road {
         result
     }
 
-    /// Calculates and sets the center_line from reference_line, reference_line_placement
-    /// (and TODO trim_start, trim_end).
```

Comment:
I took away the `TODO` cause I'm not sure it makes sense. `update_geometry.rs` is the place that applies the trim. What do you think?